src/ListComponent.js

```diff
@@ -32,6 +35,21 @@ const ListComponent = (props) => {
         /* add here */
     } 
 
+    const handleChange = (id)=>{
```

Comment:
Is this needed with the suggested changes? Honestly not sure, might be able to get rid of it?